components/ContactForm.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+export default function ContactForm(props) {
+  return (
+    <form
+      className="form rounded-lg bg-white p-4 flex flex-col"
+      onSubmit={props.onSubmit}
+    >
+      <label htmlFor="name" className="text-sm text-gray-600 mx-4">
+        {" "}
+        Your Name
+      </label>
+      <input
+        type="text"
+        className="font-light rounded-md border focus:outline-none py-2 mt-2 px-1 mx-4 focus:ring-2 focus:border-none ring-blue-500"
+        name="name"
+      />
```

Comment:
Might be fun to extract this as an `Input` or `TextInput` component down the road, but out of scope for this PR I think